corehq/toggles.py

```diff
@@ -156,6 +157,19 @@ def enabled(self, item, namespace=Ellipsis):
         ):
             # Don't even bother looking it up in the cache
             return False
+
+        # item should be a string
+        _soft_assert = soft_assert(notify_admins=False, send_to_ops=True)
```

Comment:
This part needs to be moved outside the function - usually to the top of the file.  That way `soft_assert` is only called once, when the code is first loaded.  That's how it tracks the number of hits for the exponential backoff.

Reply:
Ah, gotcha. Fix coming.